src/app/kbv/controllers/single-amount-question.js

```diff
@@ -52,7 +53,18 @@ class SingleAmountQuestionController extends BaseController {
         return callback(err);
       }
 
-      const userInput = req.body[req.session.question.questionKey];
+      let userInput = req.body[req.session.question.questionKey];
+      userInput = userInput && fields.stripSpaces(userInput);
+
+      const keysToStripDecimal = [
+        "rti-p60-earnings-above-pt",
+        "rti-p60-postgraduate-loan-deductions",
+        "rti-p60-student-loan-deductions",
+      ];
```

Comment:
Just a thought, it would be nice to keep this kind of field-specific logic out of the controller - could we do something like add our own property `stripDecimals: true` in fields.js and find the fields that way? (if this doesn't break things)